### PR TITLE
Fix IEEE after site change

### PR DESCRIPTION
**Pull request by adam3smith**
And asyncify
There's more clean-up and re-writing one could do here, but I figured getting a fix out quickly would be good and that's all I'll have time for this weekend.

### CODE REVIEW

File: IEEE Xplore.js [hunk 1]
```diff
@@ -151,98 +148,101 @@ function scrape(doc, url) {
...
-		// remove empty tag - we can take this out once empty tags are ignored
-		text = text.replace(/(keywords=\{.+);\}/, "$1}");
-		var earlyaccess = false;
-		if (text.search(/^@null/) != -1) {
-			earlyaccess = true;
-			text = text.replace(/^@null/, "@article");
+	let bibtexURL = "/rest/search/citation/format?recordIds=" + arnumber + "&fromPage=&citations-format=citation-abstract&download-format=download-bibtex";
+	Z.debug(bibtexURL);
+	// metadata is downloaded in a JSON data field
+	let bibtex = await requestJSON(bibtexURL);
```
[the remark this answers is not in the document]
Comment (dstillman) [reply IEEE Xplore.js:154]: Unfortunately it seems passing a Referer won't work in Chrome for now due to https://github.com/zotero/zotero-connectors/issues/416. Let's get this version out, and then maybe we'll need a fallback for Chrome users.

[the remark this answers is not in the document]
Comment (dstillman) [reply IEEE Xplore.js]: (Though I'm not sure how this was working for @adam3smith without `Referer`… Only intermittently enforced?)

**Comment by dstillman** [comment]
I've pushed this for now, but we'll need to figure out something for Chrome/Edge users where missing Referer is resulting in empty response.

**Comment by adam3smith** [comment]
FWIW, this works for me in both Chrome and Edge -- I did get an empty reponse from Edge once when I tried the unproxied URL to get the bibtex, so that might be a clue?

**Pull request merged**